tests/returntypes/jax/test_jax_qnode_new.py

```diff
@@ -1391,3 +1388,687 @@ def circuit(data, weights, coeffs):
     #             -np.sin(d[1] + w[1]),
     #         ]
     #         assert np.allclose(grad2_w_c, expected, atol=0.1)
+
+
+qubit_device_and_diff_method_and_mode = [
+    ["default.qubit", "backprop", "forward"],
+    ["default.qubit", "finite-diff", "backward"],
+    ["default.qubit", "parameter-shift", "backward"],
+    ["default.qubit", "adjoint", "forward"],
+    ["default.qubit", "adjoint", "backward"],
+]
+
+jacobian_fn = [jax.jacobian, jax.jacrev, jax.jacfwd]
+
+
+@pytest.mark.parametrize("dev_name,diff_method,mode", qubit_device_and_diff_method_and_mode)
+@pytest.mark.parametrize("shots", [None, 10000])
+class TestReturn:
```

Comment:
This class was deleted in https://github.com/PennyLaneAI/pennylane/pull/3235 but it should not have been the case.

Reply:
Oh boy, thank you for catching this! :pray: